app/src/main/res/values/strings.xml

```diff
@@ -3,5 +3,7 @@
 
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="map_fragment_pager_label">Map</string>
+    <string name="list_fragment_pager_label">List</string>
     <string name="art_object_detail_image">Art object detail image</string>
```

Comment:
It was a content description of the removed detail image. It looks that this string is unused now.